src/test/java/com/google/javascript/cl2dts/DeclarationGeneratorTests.java

```diff
@@ -67,7 +67,7 @@ static File getGoldenFile(final File input) {
   static String getTestFileText(final File input) throws IOException {
     String text = Files.asCharSource(input, Charsets.UTF_8).read();
     // Strip test comments in .d.ts golden files starting with '//!!'.
-    return text.replaceAll("^\\s*//!!.*\\n", "");
+    return text.replaceAll("[ ]*//!!.*\\n", "");
```

Comment:
I think `\\s` was correct, the problem you ran into is probably the anchoring. If you switch the regexp to multiline mode using `"(?m)^\\s*//!!.*\\n"` this should be good.

Though actually I cannot see a reason not to strip the comments on all trailing lines, even if there was non-whitespace before.